README.md

```diff
@@ -4,10 +4,6 @@ This is the Kids First RNA-Seq pipeline, which includes fusion and expression de
 
 ![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
 
-
-
-
-
 ## Introduction
 This pipeline utilizes cutadapt to trim adapters from the raw reads, if necessary, and passes the reads to STAR for alignment.
 The alignment output is used by RSEM for gene expression abundance estimation.
```

Comment:
Similar to joint JG comment, perhaps add a link to the public app here, and in the doc of the cwl, add a link to the git release

Reply:
Similar to the JG workflow my solution was to add a link to the public app in the readme as you did but I avoided adding it to the doc and instead made it an sbg:links label.